test/built-ins/DataView/prototype/setFloat32/index-check-before-value-conversion.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2015 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es7id: 24.2.1.2
```

Comment:
I think the frontmatter should mention `DataView.prototype.setFloat32` instead of `SetViewValue`.